openfl/component/ca/ca.py

```diff
@@ -27,6 +28,26 @@
 CA_CONFIG_JSON = Path('config/ca.json')
 
 
+def get_system_and_architecture():
+    """Get system and architecture of machine."""
+    uname_res = platform.uname()
+    system = uname_res.system.lower()
+
+    architecture_aliases = {
+        ('x86_64', ): 'amd64',
+        ('armv6l', ): 'armv6',
+        ('armv7l', ): 'armv7',
+        ('aarch64', ): 'arm64'
+    }
+    architecture = uname_res.machine
+    for aliases in architecture_aliases:
+        if architecture in aliases:
```

Comment:
If change `architecture_aliases` typing to Dict[str, str], we can compare them using `architecture == aliases`

Reply:
Changed to Dict[str, str] and `architecture == aliases`